pkg/compiler/compilation_document.go

```diff
@@ -104,8 +112,16 @@ func postCompileHook(dir string, file core.File, hook string, log *zap.SugaredLo
 	}
 
 	cmd := exec.Command(hookSegments[0], args...)
-	log.Infof(`running post-output hook: %s`, strings.Join(cmd.Args, " "))
 	cmd.Dir = dir
+
+	quotedArgs := cmd.Args
+	for i, arg := range quotedArgs {
+		if !unquotedCharsRe.MatchString(arg) {
+			quotedArgs[i] = fmt.Sprintf(`%#v`, arg)
```

Comment:
What does `%#v` do with strings? I'd expect something like `%q` here, or just using `strconv.Quote`.

Reply:
> %#v	a Go-syntax representation of the value

`%q` also works, I just didn't read that far down the docs. ;-) `strconv.Quote` is also fine, I just didn't know about it.